config/environment.js

```diff
@@ -226,9 +226,9 @@ module.exports = function(environment) {
             },
         },
         signUpPolicy: {
-            termsLink: `${POLICY_URL_PREFIX}TERMS_OF_USE.md`,
-            privacyPolicyLink: `${POLICY_URL_PREFIX}PRIVACY_POLICY.md`,
-            cookiesLink: `${POLICY_URL_PREFIX}PRIVACY_POLICY.md#f-cookies`,
+            termsLink: 'https://osf.io/terms_of_use/',
+            privacyPolicyLink: 'https://osf.io/privacy_policy/',
+            cookiesLink: 'https://osf.io/privacy_policy/#f-cookies',
```

Comment:
`#f-cookies` does not appear to be a valid anchor after redirect

Reply:
Currently there's an nginx rewrite in place that doesn't care about url paths, but that will be removed with policy updates rolling out later today, and the PP and TOU will be rendered in-app.

![Screenshot 2025-01-10 at 11 17 15](https://github.com/user-attachments/assets/282b20b5-6f2c-48a8-9b26-0fa7a3c2e424)

Reply:
oh, nice!